templates/agenda-for-all-hands-call.md

```diff
@@ -20,4 +20,8 @@
 - Demos
 - Moderator/notetaker for next time:
  * Moderator: TBD
- * Notetaker: TBD
\ No newline at end of file
+ * Notetaker: TBD
+
+<!-- After each call, it is the responsibility of the notetaker to save the last
+version of the notes in a file in ipfs/pm/meeting-notes, by opening a branch and
+submitting a PR. -->
```

Comment:
This could be also automated by the sprint-helper-bot (just do a round robin and PR all of the notes in the hackpads)